backend/fittoring/src/main/java/fittoring/mentoring/presentation/api/ReviewController.java

```diff
@@ -0,0 +1,37 @@
+package fittoring.mentoring.presentation.api;
+
+import fittoring.config.auth.Login;
+import fittoring.config.auth.LoginInfo;
+import fittoring.mentoring.business.service.ReviewService;
+import fittoring.mentoring.business.service.dto.ReviewCreateDto;
+import fittoring.mentoring.presentation.dto.ReviewCreateRequest;
+import fittoring.mentoring.presentation.dto.ReviewCreateResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class ReviewController {
+
+    private final ReviewService reviewService;
+
+    @PostMapping("reviews")
```

Comment:
p1: URL 절대경로 처리 해주세요~

```suggestion
    @PostMapping("/reviews")
```

Reply:
이걸 빼먹었네요 감사합니다!